doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -114,12 +114,14 @@ Rewriting Rules
 
 Header rewriting rules consist of zero or more `Conditions`_ followed by one or
 more `Operators`_. Conditions are used to limit the requests which will be
-affected by the operator(s). Additionally, both conditions and operators may
-have flags which modify their behavior.
+affected by the operator(s), and define when the ruleset is run during the
```

Comment:
I think we don't want this comma. (List of two things Conditions are used for.)

Reply:
@mlibbey Can you please make the minor change and we can get this PR merged in?